recipes/repository.rb

```diff
@@ -102,7 +102,10 @@
     # Import key if fingerprint matches
     execute 'rpm-import datadog key e09422b3' do
       command "rpm --import #{new_key_local_path}"
-      only_if "gpg --dry-run --quiet --with-fingerprint #{new_key_local_path} | grep 'A4C0 B90D 7443 CF6E 4E8A  A341 F106 8E14 E094 22B3'"
+      only_if {
```

Comment:
There is a warning on this:

```
* execute[rpm-import datadog key 4172a230] action run[2019-07-23T17:28:16+00:00] WARN: only_if block for execute[rpm-import datadog key 4172a230] returned "      Key fingerprint = 60A3 89A4 4A0C 32BA E3C0  3F0B 069B 56F5 4172 A230\n", did you mean to run a command? If so use 'only_if "      Key fingerprint = 60A3 89A4 4A0C 32BA E3C0  3F0B 069B 56F5 4172 A230\n"' in your code.
       [2019-07-23T17:28:16+00:00] WARN: only_if block for execute[rpm-import datadog key 4172a230] returned "      Key fingerprint = 60A3 89A4 4A0C 32BA E3C0  3F0B 069B 56F5 4172 A230\n", did you mean to run a command? If so use 'only_if "      Key fingerprint = 60A3 89A4 4A0C 32BA E3C0  3F0B 069B 56F5 4172 A230\n"' in your code.
```

We should probably use:
```
only_if "gpg --dry-run --quiet --with-fingerprint #{new_key_local_path} | grep 'A4C0 B90D 7443 CF6E 4E8A  A341 F106 8E14 E094 22B3' || gpg --dry-run --import --import-options import-show #{new_key_local_path} | grep 'A4C0B90D7443CF6E4E8AA341F1068E14E09422B3'"
```

Wdyt?

Reply:
I wanted to avoid putting everything on one line (because that's a super long line), but since this causes a warning I'll change it